src/components/Book.js

```diff
@@ -3,42 +3,44 @@ import PropTypes from 'prop-types';
 import { Button } from 'react-bootstrap';
 import './Book.css';
 
-const Book = ({ title, author }) => (
+const Book = ({
+  item_id, title, author, category, onRemove,
+}) => (
   <div className="book-container p-4 border">
     <div>
       <ul className="list-unstyled">
         <li>
           <div>
-            <div className="book-data d-flex justify-content-between">
+            <div className="book-data d-flex align-items-center justify-content-between ">
               <div className="book-details">
                 <div className="d-flex flex-column">
-                  <span>Action</span>
-                  <span>{title}</span>
+                  <span className="fw-bold gray">{category}</span>
+                  <span className="fw-bold title">{title}</span>
                   <span className="text-primary">{author}</span>
                 </div>
                 <div className="book-buttons-container mt-3">
                   <Button variant="light  text-primary">Comments</Button>
                   <span className="mx-1">|</span>
-                  <Button variant="light text-primary">Remove</Button>
+                  <Button variant="light text-primary" onClick={() => onRemove(item_id)}>
+                    Remove
+                  </Button>
```

Comment:
> Styling is **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/)

> Your final design should match Zeplin's design (this includes all elements + the same styling)

- You have done great so far, :+1: :100: But there are some slight changes you need to make before moving to the next milestone. :+1: :+1: 
- [ ] Looking at the project requirement as I pointed out above, it says your design need to be **Exactly** like the given design, so looking at the screenshot below, you can see that your texts are not properly aligned like the one on the provided design. I'll suggest you update his styling. :+1: :+1: 

- - Your design:
![Selection_092](https://github.com/kit0-0/bookstore-react-redux/assets/107632780/4eaa3e6e-f714-429b-b71d-114a5a24db98)

- - Provided design:
![Selection_090](https://github.com/kit0-0/bookstore-react-redux/assets/107632780/dd4a4d2a-7501-48a9-b7b8-d4353d71be87)